### ISSUE TITLE
Repository name must be lowercase

### ISSUE THREAD

**Issue by Acelator** [github/super-linter#190]
I'm trying to configurate Super-linter in my Github Repo [(This one)](https://github.com/acelator/viper) but I get a error that says 
`invalid reference format: repository name must be lowercase`

I have changed my repo name to delete "_" and Capital leters but it still gave me the same error. I have even changed the name set in my package.json. I don't know which name isn't lowercase

**Comment by Hanse00** [comment]
Hey @Acelator!

I copied your linter.yaml file, found at: https://github.com/Acelator/viper/blob/d7481650379a7a0d28db228df7e59c02ed82debf/.github/workflows/linter.yml, and tried to use it in my own test project.

I do indeed get the same error message:
```
6 /usr/bin/docker: invalid reference format: repository name must be lowercase.
7 See '/usr/bin/docker run --help'.
```

It's worth noting this error is coming from docker, as it tries to start the linter, not the linter itself.
I did however not have any issues in my project when using my own linter configuration file, based on the one found in the [repository README file](https://github.com/github/super-linter#example-connecting-github-action-workflow).

So I suspect the issue may lie with your configuration file.

Two little things I found in your configuration that stand out:
* You have set the ENV variable: `ACTIONS_ RUNNER_DEBUG: true` however there is an erroneous space between ACTIONS_ and RUNNER.
* You have the action configured to run on all pushes, and also on pull requests. This is redundant, as when you open a pull request, it will already run for the commits in that pull request in the first place. So the linter actually runs twice on each pull request in the configuration.

These may however not be the cause of the error message, I'm frankly not quite sure. I'd recommend you copy that configuration from the README, and modify it to suit your needs. If that doesn't work, let us know 😄
